openSUSEway.spec

```diff
@@ -184,6 +185,7 @@ install -D -p -m 644 .config/swaync/style.css %{buildroot}%{_sysconfdir}/sway/sw
 # bug #1176195, don't force enviroment, cleaning up old installations
 test -e %{_sysconfdir}/profile.d/openSUSEway.sh && rm %{_sysconfdir}/profile.d/openSUSEway.sh || true
 test -e %{_prefix}/lib/environment.d/50-openSUSEway.conf && rm %{_prefix}/lib/environment.d/50-openSUSEway.conf || true
+test -e %{_prefix}/lib/environment.d/30-windows.conf && rm %{_prefix}/lib/environment.d/30-windows.conf || true
```

Comment:
you don't need this. That was some old design, your files are new and in a proper dir already